### PR TITLE
initial documentation specifically for development

### PR DESCRIPTION
**Pull request by npdoty**
addresses #449.

### CODE REVIEW

**Comment by Christovis** [comment]
Maybe we could also add a bit regarding the creation of an Auth session for 3GPP mailing lists scraping? Or is such detail better in the wiki?

**Comment by npdoty** [comment]
> Maybe we could also add a bit regarding the creation of an Auth session for 3GPP mailing lists scraping? Or is such detail better in the wiki?

That should definitely be documented! But I think probably best not to have data-source-site-specific documentation on the main README. (I actually think we could trim a lot; I don't know that the Git or ietfdata sections should be there, beyond simple links to documentation elsewhere.)

File: README.md [hunk 1]
```diff
@@ -74,18 +74,27 @@ The script takes an argument, the working group acronym
...
+
+### Unit tests
+
+To run the automated unit tests, use: `pytest tests/unit`.
+
+Our current goal is code coverage of **60%**. Add new unit tests within `tests/unit`. Unit tests run quickly, without relying on network requests.
+
+### Documentation
+
+Docstrings are preferred, so that auto-generated web-based documentation will be possible ([#412](https://github.com/datactive/bigbang/issues/412)). You can follow the [Google style guide for docstrings](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings).
```
[the remark this answers is not in the document]
Comment (sbenthall) [reply README.md:93]: At the point where we automate the rendering of the docstrings to HTML in something like ReadTheDocs, we'll have to pick one or the other I think.

File: README.md [hunk 2]
```diff
@@ -74,18 +74,27 @@ The script takes an argument, the working group acronym
...
-
-## Git
+### Git
 
 BigBang can also be used to analyze data from Git repositories.
 
 Documentation on this feature can be found [here](https://github.com/datactive/bigbang/blob/master/git-readme.md).
 
-## Unit tests
+## Development
```
[the remark this answers is not in the document]
Comment (sbenthall) [reply README.md:83]: That would be good. Could you make a separate PR for that? I don't think it should block this PR.

**Pull request merged**